packages/quantic/scripts/build/create-package.ts

```diff
@@ -0,0 +1,199 @@
+    },
+  };
+}
+
+async function authorizePublishingOrg(log: StepLogger, options: Options) {
+  log(`Authorizing user: ${options.jwt.username}`);
+  await authorizeOrg({
+    username: options.jwt.username,
+    isScratchOrg: false,
+    jwtClientId: options.jwt.clientId,
+    jwtKeyFile: options.jwt.keyFile,
+  });
+  log('Authorization successful');
+}
+
+async function removeTranslations(log: StepLogger) {
+  const translationDir = path.resolve('force-app/main/translations/');
+  await fs.promises
+    .readdir(translationDir)
+    .then((f) =>
+      Promise.all(
+        f.map((e) => fs.promises.unlink(path.resolve(translationDir, e)))
+      )
+    );
+  log('Removed translation files');
+}
+
+async function createPackage(
+  log: StepLogger,
+  options: Options
+): Promise<SfdxCreatePackageVersionResponse> {
+  log(`Creating Quantic package version ${options.packageVersion}...`);
+
+  const response = await sfdx.createPackageVersion({
+    packageId: options.packageId,
+    packageVersion: options.packageVersion,
+    timeout: 30,
+  });
+
+  log('Quantic package created.');
+  return response;
+}
+
+async function promotePackage(log: StepLogger, packageVersionId: string) {
+  log(`Promoting Quantic package ${packageVersionId} to released status...`);
+
+  await sfdx.promotePackageVersion({
+    packageVersionId,
+  });
+  log(`Quantic package ${packageVersionId} published.`);
+}
+
+async function createGithubRelease(
+  log: StepLogger,
+  options: Options,
+  packageVersionId: string
+) {
+  const token = process.env.GITHUB_TOKEN || '';
+  const github = new Octokit({auth: token});
+
+  const packageDetails = (await sfdx.getPackageVersionList()).result.find(
+    (pack) => pack.SubscriberPackageVersionId === packageVersionId
+  );
+  const releaseName = `${packageDetails.Package2Name}v${options.packageVersion}`;
+
+  log('Creating Github release...');
+  await github.rest.repos.createRelease({
+    owner: 'coveo',
+    repo: 'ui-kit',
+    tag_name: releaseName,
+    name: releaseName,
+    body: `Package ID: ${packageDetails.SubscriberPackageVersionId}\nInstallation URL: ${packageDetails.InstallUrl}`,
+  });
+  log('Github release created.');
+}
+
+(async function () {
+  const options = await buildOptions();
+
+  const runner = new StepsRunner();
+
+  let packageVersionId: string;
+
+  if (options.promote && (await getIsPublished())) {
+    console.info(
+      `Skipped publishing ${options.packageVersion} since this patch version is already published.`
+    );
+    return;
+  }
+
+  try {
+    runner.add(async (log) => await authorizePublishingOrg(log, options));
+    options.removeTranslations &&
+      runner.add(async (log) => await removeTranslations(log));
+    runner.add(async (log) => {
+      packageVersionId = await (
+        await createPackage(log, options)
+      ).result.SubscriberPackageVersionId;
+      !options.promote &&
+        console.log(await getMatchingPackageVersion(options.packageVersion));
```

Comment:
I guess it would make sense to add some context to the output so we know what the version number refers to.

You should use `log` instead of `console.log` so the trace is logged with the proper step information.

Reply:
I'll test it out. I just want the object to be displayed with an object structure and not stringified into one line.

Reply:
you can try `JSON.stringify(obj, null, 2)` this will pretty print the object.